test/integration/canonicalize/canonical.expected

```diff
@@ -56,6 +59,8 @@ model {
   target += normal_lpdf(x| 0, 1) + normal_lcdf(2| 0, 1)
             + normal_lccdf(3| 0, 1);
   target += sum(K);
+  target += normal_cdf(1, 2, 3);
+  target += foo_cdf(1, 2, 3);
```

Comment:
These should be
```stan
  target += normal_cdf(1| 2, 3);
  target += foo_cdf(1| 2, 3);
```
You forgot to update the canonicalizer.

Reply:
Oooo, thanks, I'll investigate

Reply:
The code that replaces `_log` functions with `_lpdf` distributions is here
https://github.com/stan-dev/stanc3/blob/4def985a806f1fdee9dcfc18187a789e673d49fa/src/frontend/Canonicalize.ml#L36-L39
You can add something there to change `_cdf` `FunApp`s to `CondDistApp`s.

Reply:
I added a think to the canonicalizer but I'm not sure the tests are right. I ran them and saw this in output:

```
File "test/integration/canonicalize/canonical.expected", line 62, characters 0-1:
 |  c ~ poisson_log(3.0);
 |  if (a) {
 |    x ~ normal(0, 1);
 |    x ~ normal_log(0, 1);
 |    x ~ normal_log(0, 1);
 |    target += normal_log_lpdf(x| 1, 2);
 |    target += std_normal_lpdf(x| );
 |  }
 |  else {
 |    x ~ exponential(1);
 |    x ~ exponential(1);
 |    target += foo_lcdf(x| 1);
 |  }
 |  target += normal_lpdf(x| 0, 1) + normal_lcdf(2| 0, 1)
 |            + normal_lccdf(3| 0, 1);
 |  target += sum(K);
-|  target += normal_cdf(1, 2, 3);
+|  target += normal_cdf(1| 2, 3);
 |  target += foo_cdf(1, 2, 3);
 |  print("target: ", target());
 |}
 |generated quantities {
 |  array[2] real y0 = {1.0, 2.0};
 |  array[3] real ts = {0.5, 1.0, 2.0};
 |  array[3, 2] real y_hat = integrate_ode(sho, y0, 0.0, ts, theta, x_r, x_i);
 |  array[3, 2] real y_hat_45 = integrate_ode_rk45(sho, y0, 0.0, ts, theta,
 |                                                 x_r, x_i);
 |  array[3, 2] real y_hat_bdf = integrate_ode_bdf(sho, y0, 0.0, ts, theta,
 |                                                 x_r, x_i);
 |  array[3, 2] real y_hat_adams = integrate_ode_adams(sho, y0, 0.0, ts, theta,
 |                                                     x_r, x_i);
 |}
```

which seems right to me but `dune promote` didn't add it to the commit.

Reply:
Try dune clean and try again.

Reply:
Dune clean worked thanks!